js/directives/lucca/user-picker.js

```diff
@@ -58,12 +74,7 @@
 				operation: "@"
 			},
 			link: function (scope, elt, attrs, ctrl) {
-				if (attrs.homonymsProperties) {
-					scope.properties = attrs.homonymsProperties.split(',');
-				}
-				else {
-					scope.properties = DEFAULT_HOMONYMS_PROPERTIES;
-				}
+				ctrl.properties = attrs.homonymsProperties ? scope.homonymsProperties : DEFAULT_HOMONYMS_PROPERTIES;
```

Comment:
I'm not sure this works here because you will have in your controller
```js
$scope.myUser = {};
$scope.myProperties = [{name:"birthDate", label:"Birth date"}];
```
and in the html
```html
<luid-user-picker ng-model="myUser" homonyms-properties="myProperties"></luid-user-picker>
```
in that case `attrs.homonymsProperties` equals the string `"myProperties"` and not the variable `$scope.myProperties` or its value